packages/amplify_auth_plugin_interface/lib/src/Session/AuthUserRequest.dart

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+import './AuthUserOptions.dart';
+
+
+class AuthUserRequest {
+  AuthUserOptions options;
```

Comment:
How does this member get initialized or used?

Reply:
It does not right now, but I'm thinking it might once there is additional work in native libraries. Just including it for future use.